addons/survey/models/survey_question.py

```diff
@@ -265,6 +265,20 @@ def _compute_is_scored_question(self):
             else:
                 question.is_scored_question = False
 
+    # ------------------------------------------------------------
+    # CRUD
+    # ------------------------------------------------------------
+
+    @api.ondelete(at_uninstall=False)
+    def _unlink_except_live_sessions_in_progress(self):
+        surveys = self.env['survey.survey'].search([])
+        running_surveys = surveys.filtered(lambda survey: survey.session_state == 'in_progress')
+        if self.survey_id in running_surveys:
```

Comment:
This is fetching more data than required for this purpose. This should work just as well, right?

```suggestion
        running_surveys = self.survey_id.filtered(lambda survey: survey.session_state == 'in_progress')
        if running_surveys:
```

Reply:
@flch-odoo requested changes are done 
please review it.